src/PKSim.Core/Repositories/IContainerParametersNotCommonForAllSpeciesRepository.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using OSPSuite.Utility.Collections;
+
+namespace PKSim.Core.Repositories
+{
+   public interface IContainerParametersNotCommonForAllSpeciesRepository : IStartableRepository<(string ContainerPath, string ParameterName, int SpeciesCount)>
+   {
+      bool UsedForAllSpecies(string containerPath, string parameterName);
```

Comment:
This is the function, which returns if a parameter in a container is used for all species or not.
@msevestre Is this sufficient? Or do you need another interface?

Reply:
I need one for the parmaeter path completely as well probably. but I can add if required. Looks good